### PR TITLE
Implementing the features that tap into the API

### PR DESCRIPTION
**Pull request by German-Cobian**
Implemented the "Refresh" button (receiving data from the API and parsing the JSON).
Implemented the form "Submit" button (sending data to the API).
Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,48 @@
...
+        const li = document.createElement('li');
+        const p = document.createElement('p');
+        p.innerHTML = `${element.user}: ${element.score}`;
+        li.appendChild(p);
+        players.appendChild(li);
+      });
+    });
+  return response;
+};
+
```
Comment (Dipeshtwis) [inline src/index.js:46]: [OPTIONAL]

- When we first load the website the recent score is empty. To give the best user experience, you can fetch all the scores when the window loads. You can call the `refreshScores` function when the window loads.

Currently it looks like this.

Screenshot from 2021-08-05 22-00-11

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @German-Cobian, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Submit button hit the POST API endpoint and post user entries and Refresh button hit the GET API endpoint causing the scores list to change. :muscle:
:+1: Used `async` and `await` with the `fetch` method to send and receive data.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [ ] Please remove the unnecessary file or put it into the `.gitignore` file so that we can avoid pushing unnecessary files into Github. Ex: `.DS_Store`


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by German-Cobian** [comment]
Included DS_Store in gitignore folder
Changed code so that scores load up of first load.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @German-Cobian , 

Great work with making all the changes requested by the previous TSE.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**